service/ssm/ssmiface/interface.go

```diff
@@ -41,7 +41,7 @@ import (
 //
 //    // Define a mock struct to be used in your unit tests of myFunc.
 //    type mockClientClient struct {
-//        ssmiface.ClientPI
```

Comment:
This for pointing this out. This issue is actually apart of the SDK's code generation. With the `feature/SmithyAPIClient` branch we're working these iface packages will be removed.